pkg/errors/cdc_errors.go

```diff
@@ -959,6 +959,10 @@ var (
 		"internal check failed, %s",
 		errors.RFCCodeText("CDC:ErrInternalCheckFailed"),
 	)
+	ErrSameUpstreamDownstream = errors.Normalize(
```

Comment:
There should have Error which indicates that configuration is incorrect, just use it should be OK, no need to define new error type

Reply:
We  need to define a standard error for Open API, which is used by user.